httpclient-parent/httpclient/src/main/java/io/yupiik/uship/httpclient/core/listener/impl/HARDumperListener.java

```diff
@@ -61,12 +62,33 @@ public class HARDumperListener implements RequestListener<HARDumperListener.Data
     private final Logger logger;
     private final Har har = new Har();
 
+    private boolean withStartedDateTime = true;
```

Comment:
What about a single toggle?

Reply:
Maybe we can use a config class like `ClientHARDDumperConfiguration`?
For the single toggle, it's can be wrap in the consumer part?

Reply:
A nested Configuration works for me while we keep api compat (keeping current constructor signature).

Not sure what you mean about consumer point.